packages/frontend/src/components/attachment/messageAttachment.tsx

```diff
@@ -217,3 +228,20 @@ export function DraftAttachment({
     )
   }
 }
+
+function getDimensions(
+  message: Pick<T.Message, 'dimensionsHeight' | 'dimensionsWidth'>
+): { height: number | undefined; width: number | undefined } {
+  // With some sanity checks, in case core couldn't determine dimensions
+  // for whatever reason, but the browser might.
+  return {
+    height:
+      message.dimensionsHeight > 0 && message.dimensionsHeight < 100_000
+        ? message.dimensionsHeight
```

Comment:
why we have a limit of 100_000 here? Would 10_000 not be enough?

Reply:
Probably. But it's just a sanity check.

Reply:
Maybe someone has 4 4K monitors stacked on top of each other on their wall 🤷‍♀️